src/lib/analyses.ts

```diff
@@ -262,12 +274,35 @@ export interface HealthIndicator {
   indication: HealthIndication
```

Comment:
How about we flatten this by removing the `HealthIndication` and the indication term? All this nesting and terminology makes the code pretty hard to follow.

```suggestion
  code: HealthIndicatorCode
  reason: string
  severity: HealthIndicatorSeverity
```

Reply:
Also, is there a reason to keep `link` optional?

Reply:
Yeah I was thinking about that but I feel this is cleaner, these all belong to the indication so we should at least be prefixing them:

`
  indicationCode: HealthIndicatorCode
  indicationReason: string
  indicationSeverity: HealthIndicatorSeverity
`

I think it is cleaner this way and gives us a better pattern with the bracketing code.

Reply:
Given the large number of interfaces and redundant terminology introduced here, I very strongly feel that the extra nesting and the subtle distinction between indicator and indication over-complicate things. It's especially confusing with `value` being a part of the indicator, but `severity` and `code` being a part of the indication. With a flat structure and fewer interfaces, there will be less to keep in mind, and I think that it can actually improve the bracketing code.

Reply:
I strongly feel that the indication is a seperate derivation of the value and the indicator type, particularly the `code`, `reason`, `severity`, and the possible future `recommendation` should all be grouped together. 

To me this is a case of "as simple as possible but no simpler".

> so we should at least be prefixing them

What did you think about this? I'm also happy to change the names around.

Reply:
Prefixing them would be better, though I still don't think that the concept of an indication is worth keeping.

Reply:
After trying it out I am not convinced the flat structure is any better and it seems to me like the only difference is we are using `indicationCode` instead of `indication.code`, I am going to go forward with the nested indication - it is a relatively local decision and we can always change it later.